File: ce/snippets/csharp/CRMV8/marketing/cs/quickcampaign1.cs
// <snippetquickcampaign1>



 List newList = new List()
 {
     ListName = "TestList", 
     CreatedFromCode = new OptionSetValue((int)ListCreatedFromCode.Account)
 };

 _newListId = _serviceProxy.Create(newList);

 for (int j = 0; j < 5; j++)
 {
     AddMemberListRequest addMemberListRequest = new AddMemberListRequest();
     addMemberListRequest.EntityId = _accountIdArray[j];
     addMemberListRequest.ListId = _newListId;
     AddMemberListResponse addMemberListResponse = 
         _serviceProxy.Execute(addMemberListRequest) as AddMemberListResponse;
 }

 Guid BOId = CreateAndRetrieveQuickCampaignForMarketingList(
     _templateLetterActivity, 
     _newListId, 
     PropagationOwnershipOptions.ListMemberOwner, 
     true);

// </snippetquickcampaign1>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/marketingvb/vb/quickcampaign1.vb
' <snippetquickcampaign1>



 Dim newList As New List() With
     {
         .ListName = "TestList",
         .CreatedFromCode =
         New OptionSetValue(CInt(Fix(ListCreatedFromCode.Account)))
     }

 _newListId = _serviceProxy.Create(newList)

 For j As Integer = 0 To 4
     Dim addMemberListRequest As New AddMemberListRequest()
     addMemberListRequest.EntityId = _accountIdArray(j)
     addMemberListRequest.ListId = _newListId
     Dim addMemberListResponse As AddMemberListResponse =
         TryCast(_serviceProxy.Execute(addMemberListRequest), 
             AddMemberListResponse)
 Next j

 Dim BOId As Guid = CreateAndRetrieveQuickCampaignForMarketingList(
     _templateLetterActivity,
     _newListId,
     PropagationOwnershipOptions.ListMemberOwner,
     True)

' </snippetquickcampaign1>